parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -95,12 +95,12 @@ $global:PrimaryScaleSetName = "{{WrapAsVariable "primaryScaleSetName"}}"
 $global:KubeClusterCIDR = "{{WrapAsParameter "kubeClusterCidr"}}"
 $global:KubeServiceCIDR = "{{WrapAsParameter "kubeServiceCidr"}}"
 $global:VNetCIDR = "{{WrapAsParameter "vnetCidr"}}"
-{{if IsKubernetesVersionGe "1.16.0"}}
+{ { if IsKubernetesVersionGe "1.16.0" } }

```

Comment:
@marosset do you think this is fine or should I undo the formatting

Reply:
I think that needs to be undone because goloang formatting might not like the spaces in between brackets